base/applications/mmc/console.c

```diff
@@ -334,6 +336,15 @@ FrameOnCommand(HWND hwnd,
             PostMessage(hwnd, WM_CLOSE, 0, 0);
             break;
 
+        case IDM_HELP_ABOUT:
+            if (AllocAndLoadString(&lpTitle, hAppInstance, IDS_APPTITLE))
+            {
+                ShellAbout(hwnd, lpTitle, NULL,
+                           LoadIcon(hAppInstance, MAKEINTRESOURCEW(IDI_MAINAPP));
```

Comment:
Should be either `LoadIcon` with `MAKEINTRESOURCE`, or `LoadIconW` with `MAKEINTRESOURCEW`